puzzles/models.py

```diff
@@ -27,12 +27,12 @@ class InvalidMetaPuzzleError(PuzzleModelError):
 
 
 class Puzzle(models.Model):
-    name = models.CharField(max_length=80, unique=True)
+    name = models.CharField(max_length=80)
     hunt = models.ForeignKey('hunts.Hunt', on_delete=models.CASCADE, related_name='puzzles')
     url = models.URLField(blank=True)
 
-    sheet = models.URLField(default='', unique=True)
-    channel = models.CharField(max_length=128, default='', unique=True)
+    sheet = models.URLField(default='', unique=True, null=True)
+    channel = models.CharField(max_length=128, default='', unique=True, null=True)
```

Comment:
Now that we are allowing null, should the default be None?

Same question for the sheet URL.

Reply:
How about having `(unique=True, null=True, blank=True)` to include proper form validation (and default semantics) as described here? https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.Field.null

Reply:
Seems like the right thing is both changing `default=None` (an empty string default violates the uniqueness constraint anyway) and adding `blank=True`(not sure when this would come up but maybe in the admin console)